msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -393,11 +397,11 @@ public Authority getDefaultAuthority() {
         }
     }
 
-    private void checkManifestPermissions(){
-        if(this.handleNullTaskAffinity){
+    private void checkManifestPermissions() {
+        if (this.handleNullTaskAffinity) {
```

Comment:
handleNullTaskAffinity is a Boolean - probably would like to either convert it to a boolean or handle the potential for a null

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1357/commits/36700831510bbbe5339b7b97602a4d43d1320c83